Services/src/main/java/ev/projects/services/DocumentService.java

```diff
@@ -1,15 +1,17 @@
-package ev.projects.webapp.services;
+package ev.projects.services;
 
 import ev.projects.models.Document;
-import ev.projects.webapp.repositories.IDocumentRepository;
+import ev.projects.repositories.IDocumentRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
 
 @Service
+@ComponentScan({"ev.projects.repositories"})
```

Comment:
The component scan shouldn't be declared in service implementation.
I'd suggest creating a @Configuration class and put the @ComponentScan there.

Reply:
Should be solved in 5da3e6f.